.github/workflows/wasm-light-clients.yml

```diff
@@ -0,0 +1,60 @@
+name: Wasm Light Clients
+# This workflow runs when a PR is opened that targets code that is part of the 08-wasm-light-clients
+on:
+  merge_group:
+  pull_request:
+  push:
+    branches:
+      - main
+      - feat/ibc-eureka # TODO: Remove before merging to main
```

Comment:
This PR should have it's own feature branch and not be merged directly to feat/ibc-eureka in my opinion

Reply:
> If possible to get things merged in smaller increments, it will be easier than having to maintain yet another feature branch. I would prefer to avoid another branch if smaller PRs can do the job

Having been dealing with multiple feature branches for a while now, I 100% agree!